metricbeat/helper/sql/sql.go

```diff
@@ -0,0 +1,187 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/pkg/errors"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Db struct {
```

Comment:
`DbClient` is maybe a more self-explanatory name?
Also does this struct need to be exported? Could it be just `dbClient` ?

Reply:
Yeah, I think I'll change the name. This code comes directly from the `sql` module and I wanted to maintain as many things as possible like the original code. But if two people (you and me) think that it may have a better name... I didn't change it before because this code has passed already a review without complains about it :smile:

Reply:
Ah! BTW, it must be exported as it's used by every Metricset which takes advantage of it

Reply:
👍